README.md

```diff
@@ -8,9 +8,11 @@ IKE (Interactive Knowledge Extraction)
 [Live Demo](http://ike.allenai.org/)
 
 ## Run Locally
-1. Run `sbt`
-2. Enter the `reStart` command
-3. Open http://localhost:8080 in a browser
+1. Install Postgres SQL locally and create a database for use by IKE. IKE needs this to store the tables you create.
```

Comment:
Yeah I tried `PostgresSql` and `Postgres Sql` -- neither looked right, thanks  :)